man/systemd-oomd.service.xml

```diff
@@ -127,6 +127,24 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>Signals</title>
+
+    <variablelist>
+      <varlistentry>
+        <term><constant>SIGHUP</constant></term>
+
+        <listitem><para>Upon reception of the <constant>SIGHUP</constant> process signal
+        <command>systemd-oomd</command> will reload its configuration file.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <xi:include href="standard-options.xml" xpointer="help" />
+      <xi:include href="standard-options.xml" xpointer="version" />
```

Comment:
these 2 lines should not be added, they are duplicated from above

Reply:
I am assuming you are referring to the standard-options.xml lines, not the actual description? Omitted.